### PR TITLE
[DNM] [search] Statistics events to check old format usage.

### PR DESCRIPTION
**Pull request by mpimenov**
@ygorshenin @burivuh

### CODE REVIEW

**Comment by mpimenov** [comment]
On second thought, probably Downloader_Map_list is enough.

File: map/framework.cpp [hunk 1]
```diff
@@ -685,11 +686,15 @@ void Framework::RegisterAllMaps()
...
-    minFormat = min(minFormat, static_cast<int>(id.GetInfo()->m_version.GetFormat()));
+    auto const format = id.GetInfo()->m_version.GetFormat();
+    minFormat = min(minFormat, static_cast<int>(format));
     if (needStatisticsUpdate)
     {
       listRegisteredMaps << localFile->GetCountryName() << ":" << id.GetInfo()->GetVersion() << ";";
     }
+
+    version::MwmTraits mwmTraits(format);
+    alohalytics::LogEvent("SearchIndexFormat", DebugPrint(mwmTraits.GetSearchIndexFormat()));
```
Comment (ygorshenin) [inline map/framework.cpp]: Please, don't use DebugPrint() for aloha events! You need to implement your own serialization.

**Comment by mapsmetest** [comment]
Jenkins asks: can anyone have a look at this request?
If you want Jenkins to compile and test this, write "test this please" or JT_TP (without the underscore in the middle).

**Comment by mpimenov** [comment]
We've removed the support for old format in #7157.
1. As far as I understand, we do not officially support mwms that were issued earlier than a half-year before the latest version.
2. It looks like the required information can be gathered from other events.

**Pull request closed without merging**